smb/src/main/java/ch/cyberduck/core/smb/SMBCopyFeature.java

```diff
@@ -44,31 +42,19 @@ public Path copy(Path source, Path target, TransferStatus status, ConnectionCall
             return target;
         }
         catch(SMBApiException e) {
-            throw new SmbExceptionMappingService().map(e);
+            throw new SMBExceptionMappingService().map(e);
```

Comment:
Replace with parameters `"Cannot copy {0}", e, source`

```suggestion
            throw new BackgroundException("Cannot copy {0}", e, source);
```